plugins/modules/redis_info.py

```diff
@@ -219,7 +256,12 @@ def main():
         module.fail_json(msg="unable to connect to database: %s" % to_native(e), exception=traceback.format_exc())
 
     info = client.info()
-    module.exit_json(changed=False, info=info)
+
+    cluster_info = dict()
+    if cluster:
+        cluster_info = client.execute_command('CLUSTER INFO')
+
+    module.exit_json(changed=False, info=info, cluster_info=cluster_info)
```

Comment:
How about doing it this way:
```suggestion
    result = dict(changed=False, info=info)

    if cluster:
        result['cluster_info'] = client.execute_command('CLUSTER INFO')

    module.exit_json(**result)
```
Then `cluster_info` is only returned if `cluster=true` is provided.